internal/cmd/base/servers.go

```diff
@@ -462,16 +462,34 @@ func (b *Server) ConnectToDatabase(dialect string) error {
 	return nil
 }
 
-func (b *Server) CreateDevDatabase(ctx context.Context, dialect string, opt ...Option) error {
-	opts := getOpts(opt...)
+func (b *Server) CreateDevDatabase(ctx context.Context, opt ...Option) error {
 
-	var container, url string
+	var container, url, dialect string
 	var err error
 	var c func() error
 
+	opts := getOpts(opt...)
+
+	// opts.WithContainerImage could have repo:tag, so split it and only
+	// assign the repo to dialect
+	switch opts.withContainerImage {
+	case "postgres":
+		dialect = "postgres"
+	default:
+		separated := strings.Split(opts.withContainerImage, ":")
+		dialect = separated[0]
+	}
+
+	if opts.withContainerImage == "" {
```

Comment:
Seems we can drop this check and just have the above switch is we add `withContainerImage: "postgres"` to `getDefaultOptions`

Reply:
🤦‍♀️🤦‍♀️🤦‍♀️🤦‍♀️